### PR TITLE
Added very basic google OAuth

### PR DESCRIPTION
**Pull request by Azuolas3**
Added very basic google OAuth functionality, which on login logs to console a JWT token (encoded JSON text). Works only for predetermined users/google accounts.

### CODE REVIEW

File: ClientApp/src/setupProxy.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ const target = env.ASPNETCORE_HTTPS_PORT ? `https://localhost:${env.ASPNETCORE_H
 
 const context =  [
   "/weatherforecast",
-  "/helloback"
+  "/helloback",
+  "/login"
```
Comment (onlyidev) [inline ClientApp/src/setupProxy.js:10]: There is already a React route to this link. For communication with the backend please set up a different route. Currently when trying to access /login user gets 404

**Review by onlyidev — CHANGES_REQUESTED** [review]
For future: there is no need to fork the repo as we all have write access to the main repository.

File: ClientApp/src/setupProxy.js (hunk 1 above)
Comment (Azuolas3) [reply ClientApp/src/setupProxy.js]: Fixed (hopefully)

**Comment by Azuolas3** [comment]
> For future: there is no need to fork the repo as we all have write access to the main repository.

I thought so too but for some reason I couldn't push through git bash or VSCode. Since VSCode offered to fork it and then push instead so I just did that

**Review by onlyidev — APPROVED** [review]
Will look into why pushing into main repo was not allowed

**Pull request merged**